cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,153 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 5.1.0 <http://docs.cask.co/cdap/5.1.0/index.html>`__
+=============================================================
+
+- :cask-issue:`CDAP-13566` - Improved runtime monitoring (which fetches program states, metadata and logs) of remotely launched programs from the CDAP Master by using dynamic port forwarding instead of HTTPS for communication.
+
+- :cask-issue:`CDAP-13977` - Removed duplicate classes to reduce the size of the sandbox by a couple hundred megabytes
+
+- :cask-issue:`CDAP-13991` - Remove dependencies from Tracker application
+
+- :cask-issue:`CDAP-14108` - Removed old and buggy dynamic spark plugins
+
+- :cask-issue:`CDAP-14456` - Dropped support for MapR 4.1.
+
+- :cask-issue:`CDAP-14461` - Added cdap-env.sh to allow configuring jvm options for sandbox.
+
+- :cask-issue:`CDAP-14005` - Added capability to emit Field Level Lineage in DataPrep Directives
```

Comment:
this is not done. Should remove. Unless it only talks about Wrangler transform, in which case we should specify that this is Wrangler Transform only.